src/commands/package.ts

```diff
@@ -107,9 +108,32 @@ export default class Package extends Command {
           execa('npm', ['install', '--production', `--prefix=${outputDir}`, ...(platform() === 'win32' ? ['--force'] : [])], {
             stdio: flags.verbose ? 'inherit' : 'ignore'
           }).catch(e => this.error(e, { exit: 10 }))
+      },
+      {
+        title: 'Check the SAP Cloud SDK dependencies',
+        task: async () => {
+          const { dependencies, devDependencies } = await parsePackageJson(projectDir);
+          checkOldDependencies(dependencies);
+          checkOldDependencies(devDependencies);
+        }
       }
     ]);
 
     await tasks.run();
+    this.printSuccessMessage();
+  }
+
+  private printSuccessMessage() {
+    const warnings = getWarnings();
+    const body = [
+      '🚀 Please migrate to new packages.',
+      'Please find how to migrate here:',
+      'https://github.com/SAP/cloud-sdk/blob/master/docs/how-to-switch-to-os-cloud-sdk.md'
+    ];
+    if (warnings) {
```

Comment:
should the body not only be shown if the warnings contain at least on line like 
```Old SAP Cloud SDK: ${oldDependency} is detected.```.
If the warnings are related to some other issue the comment on  migration could be irrelevant.